graphql_api/types/repository/repository.graphql

```diff
@@ -1,15 +1,28 @@
+# Repository is a named collection of files uploaded
+# to a provider (e.g., github)
 type Repository {
+  # isFirstPullRequest fetches whether there is only 1 pull request (may be
+  # helpful for comparison graphs)
   isFirstPullRequest: Boolean!
+  # name fetches the repository's human readable identifier
   name: String!
+  # active fetches the repo's active status as designated by provider
   active: Boolean!
+  # activated fetches the repo's Codecov activation status
   activated: Boolean!
+  # private fetches the repo's public/private status as designated at provider
   private: Boolean!
-  coverage: Float
-  coverageSha: String
-  hits: Int
-  misses: Int
-  lines: Int
```

Comment:
Shown as deleted in this diff for illustrative purposes. In practice, we plan to delete the old fields only after everything is migrated over to the new fields and stable.

Reply:
Will you be duplicating the resolvers during the migration process as well? Ideally there would be a way to bind both the new and old schemas to the same resolvers at the same time so that the Python codes don't get duped.